lib/commands/create_command.dart

```diff
@@ -9,7 +9,7 @@ import 'package:io/ansi.dart' as ansi;
 import 'package:mason/mason.dart';
 import 'package:process_run/process_run.dart';
 
-Future<void> createCommand(ArgResults command) async {
+Future<void> createCommand(ArgResults command, Logger logger) async {
```

Comment:
for a followup, I would envision we creating some class that wraps the ArgResults and Logger to pass around as a unit, and then the "utils" methods would live on this class as well
something like "CLI  context", not sure what is a good name
but as I said - this is def a followup!